tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -135,6 +135,16 @@ static cl::opt<bool> SPIRVGenKernelArgNameMD(
     cl::desc("Enable generating OpenCL kernel argument name "
              "metadata"));
 
+static cl::opt<SPIRV::ExtInst>
+    ExtInst("spirv-ext-inst",
+            cl::desc("Specify external instructions to use when "
```

Comment:
not sure whether here should be 'an' or 'the'
```suggestion
            cl::desc("Specify the extended instruction set to use when "
```

Reply:
Done.